shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -205,12 +212,14 @@ export default {
           return '';
         }
         if ( this.subType ) {
+          // if driver type has a custom form component, don't load an ember page
+          if (this.selectedSubType.component) {
```

Comment:
Nice catch/sorry you had to debug that...I've updated the logic around subtypes a bit so that if an extension provisioner doesn't have the rke2 'group' set its visibility isn't dependent on the rke1/rke2 toggle. I think I've done this in a way that the default behaviour is unchanged/we wont break some pre-existing rke2-only extension provisioner.